lib/model/migrations/20191007-01-backfill-client-audits.js

```diff
@@ -27,7 +27,7 @@ const up = (db) => new Promise((resolve, reject) => {
       getFormFields(formXml),
       traverseXml(submissionXml, [ findOne(root(), node('meta'), node('audit'))(text()) ])
     ]).then(([ fields, [ auditNode ] ]) => {
-      if (!auditNode.isDefined()) return;
+      if (auditNode.isEmpty()) return;
```

Comment:
Even though it's a simple change, I feel nervous about changing anything in a migration, since many aren't tested. Could we revert this one?

Reply:
Done.